src/pages/Product/Top/Top.js

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import './Top.scss';
+import imgA from '../../../assets/images/backimg1.jpg';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faPlus } from '@fortawesome/free-solid-svg-icons';
+import { faCaretDown } from '@fortawesome/free-solid-svg-icons';
```

Comment:
```suggestion
import { faPlus, faCaretDown } from '@fortawesome/free-solid-svg-icons';
```
해당 부분은 이렇게 줄일 수 있겠네요!